modular_citadel/code/modules/reagents/chemistry/reagents/fermi_reagents.dm

```diff
@@ -184,7 +184,7 @@
 //Writen by Trilby!! Embellsished a little by me.
 
 /datum/reagent/fermi/nanite_b_gone
-	name = "Naninte bain"
+	name = "Naninte bane"
```

Comment:
Nanite bane.

Reply:
I dunno how I didn't see it until now

Reply:
Or maybe I should keep it as bain? Am I too attached?

Reply:
Use `bane`, `bain` is archaic and obsolete.